subgraph.yaml

```diff
@@ -232,7 +329,31 @@ dataSources:
         - event: RemoveLiquidity(indexed address,uint256[4],uint256[4],uint256)
           handler: handleCurveSUSDTransfer
       file: ./src/mappings/curve.ts
-
+  - kind: ethereum/contract
+    name: curve_susd_gauge
+    network: mainnet
+    source:
+      address: "0xA90996896660DEcC6E997655E065b23788857849"
+      abi: CurveAaveGaugeContract
```

Comment:
One repetition I keep seeing is abi file name and subgraph name. I wonder if there's a way to standardize to use 1 format and even dynamically generate name/abi from the other.

Reply:
I think there is a way to use a script to build the yaml file. It's an optimization we should do eventually.